openstack/nova/templates/hypervisors/_ironic-deployment.yaml.tpl

```diff
@@ -24,17 +24,18 @@ spec:
   template:
     metadata:
       labels:
-{{ tuple . "nova" "compute" | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
+{{ tuple . "nova" $compute_hypervisor_name | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
         name: nova-compute-{{$hypervisor.name}}
         hypervisor: "ironic"
       annotations:
+        pod.beta.kubernetes.io/hostname: nova-compute-{{$hypervisor.name}}
         configmap-etc-hash: {{ include (print .Template.BasePath "/etc-configmap.yaml") . | sha256sum }}
         configmap-ironic-etc-hash: {{ tuple . $hypervisor | include "ironic_configmap" | sha256sum }}
     spec:
       terminationGracePeriodSeconds: {{ $hypervisor.default.graceful_shutdown_timeout | default .Values.defaults.default.graceful_shutdown_timeout | add 5 }}
       containers:
-        - name: nova-compute
-          image: {{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-nova-compute:{{.Values.imageVersionNovaCompute | default .Values.imageVersion | required "Please set nova.imageVersion or similar" }}
+        - name: nova-compute-{{$hypervisor.name}}
```

Comment:
Could you leave it on `nova-compute`, otherwise you have to change two items to get the logs of a nova-compute nodes, the name of the pod, and the name of the container, instead of only one, the name of the pod.

Reply:
This has been introduced by @galkindmitrii, please resolve this